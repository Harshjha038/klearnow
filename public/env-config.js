
// window._env_ = {
//   REACT_APP_API_URL: "https://api.klearnow.com/dev-cust/v1/events",
//   REACT_APP_CHAT_KEY: "368e58ccc62de8c3013d36c5fb9bd3c1d",
//   ENVIRONMENT: "dev",
//   REACT_APP_LOGIN_URL_CUSTOMER: "https://customer-dev.klearnow.com/",
//   REACT_APP_LOGOUT_URL_CUSTOMER: "https://cob-dev.klearnow.com",
//   KX_K8S_CLUSTER_AWS_REGION: "us-west-2",
//   KX_S3_ATTACHMENT_URL: "https://kxr-attachments-public.s3.amazonaws.com",
//   REACT_APP_CUSTOMER_STRIPE_KEY: "pk_test_Wx9KxRm5f3qFThN7ODhsvLVJ00KDqbnmGf",
//   REACT_APP_SHIPMENT_MEDIA_URL: "https://api.klearnow.com/dev/v1/upload",
//   REACT_APP_MEDIA_URL: "https://api.klearnow.com/dev/v1/uploadTo",
//   CUSTOMER_MEDIA_BUCKET: "kx-customer-media",
//   TRUCKER_MEDIA_BUCKET: "kx-trucker-media-public",
//   DRAYAGE_PARTNER_DOC_BUCKET: "kx-drayage-partner-documents-dev",
//   DRAYAGE_PARTNER_MEDIA_BUCKET: "kx-drayage-partner-media-dev"
// }

window._env_ = {
    REACT_APP_API_URL: "https://api.klearnow.com/staging-cust/v1/events",
    REACT_APP_CHAT_KEY: "368e58ccc62de8c3013d36c5fb9bd3c1d",
    ENVIRONMENT: "staging",
    REACT_APP_LOGIN_URL_CUSTOMER: "https://customer-staging.klearnow.com",
    REACT_APP_LOGOUT_URL_CUSTOMER: "https://cob-staging.klearnow.com",
    REACT_APP_LOGOUT_URL_CUSTOMER_WL: "https://jdl-staging.klearnow.com",
    KX_K8S_CLUSTER_AWS_REGION: "us-east-1",
    KX_S3_ATTACHMENT_URL: "https://kxr-attachments-staging.s3.amazonaws.com",
    REACT_APP_CUSTOMER_STRIPE_KEY: "pk_test_Wx9KxRm5f3qFThN7ODhsvLVJ00KDqbnmGf",
    REACT_APP_SHIPMENT_MEDIA_URL: "https://api.klearnow.com/staging-ops/v1/upload",
    REACT_APP_MEDIA_URL: "https://api.klearnow.com/staging-ops/v1/uploadTo",
    CUSTOMER_MEDIA_BUCKET: "kx-customer-media-staging",
    TRUCKER_MEDIA_BUCKET: "kx-trucker-media-staging",
    SKIP_PREFLIGHT_CHECK: "true",
    DRAYAGE_PARTNER_DOC_BUCKET: "kx-drayage-partner-documents-staging",
    DRAYAGE_PARTNER_MEDIA_BUCKET: "kx-drayage-partner-media-staging"
  }
  
  // window._env_ = {
  //   REACT_APP_API_URL: "https://api.klearnow.com/prod-cust/v1/events",
  //   REACT_APP_CHAT_KEY: "2788340bcb148096350e02553394731f5",
  //   ENVIRONMENT: "prod",
  //   REACT_APP_LOGIN_URL_CUSTOMER: "https://customer.klearnow.com",
  //   REACT_APP_LOGOUT_URL_CUSTOMER: "https://cob.klearnow.com",
  //   REACT_APP_LOGOUT_URL_CUSTOMER_WL: "https://jdl.klearnow.com",
  //   KX_K8S_CLUSTER_AWS_REGION: "us-east-1",
  //   KX_S3_ATTACHMENT_URL: "https://kxr-attachments.s3.amazonaws.com",
  //   REACT_APP_CUSTOMER_STRIPE_KEY: "pk_live_bLgkwJVe167Ed3SbZ3ElMfTA00XnwFlLIE",
  //   REACT_APP_SHIPMENT_MEDIA_URL: "https://api.klearnow.com/prod-ops/v1/upload",
  //   REACT_APP_MEDIA_URL: "https://api.klearnow.com/prod/v1/uploadTo",
  //   CUSTOMER_MEDIA_BUCKET: "kx-customer-media-prod",
  //   TRUCKER_MEDIA_BUCKET: "kx-trucker-media-public-prod",
  //   SKIP_PREFLIGHT_CHECK: "true",
  // }
  