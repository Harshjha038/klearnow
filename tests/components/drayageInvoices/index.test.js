import React from 'react';
import renderer from 'react-test-renderer';

import '../../__mocks__/env-config';
import DrayageInvoice from '../../../src/components/drayageInvoices';

test('DrayageInvoice renders correctly', () => {
    const tree = renderer.create(
        <DrayageInvoice
        invoiceContainer = ''
        invoiceContainerNumber = {1}
        />,
    ).toJSON();
    expect(tree).toMatchSnapshot();
})