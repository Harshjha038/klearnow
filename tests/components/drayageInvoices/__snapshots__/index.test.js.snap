// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DrayageInvoice renders correctly 1`] = `
<div>
  <div>
    <div
      className="drayage-invoices-container drayage-partner"
      style={
        {
          "paddingTop": "90px",
        }
      }
    >
      <div
        className="drayage-invoice-searchbar"
      >
        <i
          className="search icon "
        />
        <input
          className="search icon drayage-invoice-input"
          onChange={[Function]}
          onKeyUp={[Function]}
          placeholder="Search Invoices"
          value=""
        />
      </div>
      <div
        className="ui fluid container"
      >
        <div
          className="drayage-title"
        >
          Invoices
        </div>
        <div
          className="drayage-country-date"
        />
        <div
          className="drayage-tabs"
        >
          <div
            className="ui attached tabular menu"
          >
            <a
              className="active item"
              onClick={[Function]}
            >
              CHARGE LIST
              <div
                className="ui circular label"
                onClick={[Function]}
              >
                0
              </div>
            </a>
            <a
              className="item"
              onClick={[Function]}
            >
              PENDING
              <div
                className="ui circular label"
                onClick={[Function]}
              >
                0
              </div>
            </a>
            <a
              className="item"
              onClick={[Function]}
            >
              PAID
              <div
                className="ui circular label label-green"
                onClick={[Function]}
              >
                0
              </div>
            </a>
            <a
              className="item"
              onClick={[Function]}
            >
              DISPUTED
              <div
                className="ui circular label label-red"
                onClick={[Function]}
              >
                0
              </div>
            </a>
          </div>
          <div
            className="ui bottom attached segment active tab tab-pane"
          >
            <div
              style={
                {
                  "display": "flow-root",
                }
              }
            />
            
            
            <div
              className="dray-partner-charge-list-open-released-btn"
            >
              <div
                className="ui buttons"
              >
                <button
                  className="ui positive button"
                  onClick={[Function]}
                  role="button"
                >
                  OPEN
                </button>
                <div
                  className="or"
                  data-text=""
                />
                <button
                  className="ui button"
                  onClick={[Function]}
                  role="button"
                >
                  GENERATED
                </button>
              </div>
            </div>
            <div
              className="ui segment custom-loader"
            >
              <div
                className="ui active transition visible dimmer"
                onClick={[Function]}
              >
                <div
                  className="content"
                >
                  <div
                    className="ui text loader"
                  >
                    Loading
                  </div>
                </div>
              </div>
              <img
                className="ui image"
                src="/images/short-paragraph.png"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div>
    <style>
      
          .common-modal.ui.fullscreen.modal{
            left: unset !important;
          }
          .common-modal  .close.icon{
                    top: auto !important;
                    position: relative !important;
                    float: right !important;
                    right: auto !important;
                    color: #0A8B8F !important;
                 }
       .ui.dimmer .common-modal .content {
                width:100% !important;
           }
    </style>
  </div>
</div>
`;
